packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -116,63 +78,209 @@ function getActiveChainIds() {
   return activeChainIds;
 }
 
-export async function getRoundsInApplicationPhase(
-  debugModeEnabled: boolean
-): Promise<RoundOverview[]> {
+const ROUNDS_QUERY = `
+query GetRounds(
+  $first: Int, 
+  $orderBy: String,
+  $orderDirection: String,
+  $where: Round_filter,
+  $currentTimestamp: String
+  ) {
+    rounds(first: $first,
+      orderBy: $orderBy,
+      orderDirection: $orderDirection,
+      where: $where
+    ) {
+      id
+      roundMetaPtr {
+        protocol
+        pointer
+      }
+      applicationsStartTime
+      applicationsEndTime
+      roundStartTime
+      roundEndTime
+      matchAmount
+      token
+      payoutStrategy {
+        id
+        strategyName
+      }
+      projects(where:{
+        status: 1
+      }) {
+        id
+      }
+    }
+}
```

Comment:
Just one query to fetch and filter rounds. We can pass these variables dynamically based on filters and sorting as well as round and application timestamps.